chain/network/src/peer_manager.rs

```diff
@@ -131,16 +129,16 @@ struct ActivePeer {
     peer_type: PeerType,
 }
 
-#[cfg(feature = "protocol_feature_routing_exchange_algorithm")]
 #[derive(Default)]
-struct IbfRoutingTableExchangeActor {
+pub struct RoutingTableActor {
```

Comment:
Please add a comment on what the actor is responsible for

Reply:
Ok

Reply:
Done. I also moved it to another file.